### PR TITLE
[Xedra Evolved] Another batch of random fixes and improvements

### PR DESCRIPTION
**Pull request by GuardianDll**
#### Summary
None
#### Purpose of change
Maintaining the mod
#### Describe the solution
See commit names (aslo ignore the one that was merged in previous PR)

### CODE REVIEW

File: data/mods/Xedra_Evolved/effects/effects.json [hunk 1]
```diff
@@ -505,7 +510,7 @@
     "type": "effect_type",
     "name": [ "Physical Instrument" ],
     "desc": [
-      "Your body is a tool of destruction.  Use it effectively.  Your attacks are faster and deadlier, and your dodge abilities is better"
+      "Your body is a tool of destruction.  <color_red>Use it effectively<color>.  Your attacks are faster and deadlier, and your dodge abilities is better"
```
Comment (TheShadowFerret) [inline data/mods/Xedra_Evolved/effects/effects.json:513]:
```suggestion
      "Your body is a tool of destruction.  <color_red>Use it effectively<color>.  Your attacks are faster and deadlier, and your dodging ability is better."
```
(or "dodging abilities are better.")

File: data/mods/Xedra_Evolved/eocs/dreamsmith.json [hunk 2]
```diff
@@ -114,11 +133,16 @@
...
-      },
-      {
-        "u_message": "You found pretty smart way to utilize a dreamdross abilities, and you can't wait to use it.",
-        "type": "info"
+        "type": "recipe",
+        "true_eocs": {
+          "id": "EOC_DREAMSMITH_SUPER_MESSAGE_POSITIVE",
+          "effect": [
+            {
+              "u_message": "You found pretty smart way to utilize a dreamdross abilities, and you can't wait to use it.",
```
Comment (TheShadowFerret) [inline data/mods/Xedra_Evolved/eocs/dreamsmith.json:141]:
```suggestion
              "u_message": "You found a pretty smart way to utilize dreamdross' abilities, and you can't wait to use it.",
```

File: data/mods/Xedra_Evolved/items/inventor/armor.json [hunk 3]
```diff
@@ -385,6 +385,61 @@
       }
     ]
   },
+  {
+    "type": "ARMOR",
+    "id": "inventor_fists",
+    "symbol": "3",
+    "color": "light_gray",
+    "name": { "str": "heavy CQC system" },
+    "description": "Heavyweight system, that start from your hands, that holds square-ish steel frame with few bumps on the end, to increase the damage output by concentrating the impact point, goes into your lower arms with thick piece of hardened steel to cover both your limb and the hydraulic system, into the upper arm with aluminum tubes for hydraulic system, and end on your back, where the main part of hydraulic servos belong.  The energy consumption is recuperated from each move, but overall weight and bulkiness of the system makes it barely possible to use with anything in your hands.",
```
Comment (TheShadowFerret) [inline data/mods/Xedra_Evolved/items/inventor/armor.json:394]:
```suggestion
    "description": "Heavyweight system, starting from your hands, that holds a square-ish steel frame with few bumps on the end, to increase damage output by concentrating the impact point.  It goes into your lower arms with thick piece of hardened steel to cover both your limbs and the hydraulic system, extends into the upper arm with aluminum tubes for hydraulics, and ends on your back, where the main part of hydraulic servos belong.  The energy consumption is recuperated from each move, but overall weight and bulkiness of the system makes it barely possible to use with anything in your hands.",
```

File: data/mods/Xedra_Evolved/items/range.json [hunk 4]
```diff
@@ -0,0 +1,135 @@
+[
+  {
+    "id": "dreamforged_longbow",
+    "looks_like": "longbow",
+    "type": "GUN",
+    "symbol": "(",
+    "color": "light_gray",
+    "name": { "str": "dreamforged bow" },
+    "description": "A six-foot bow, forged entirely from dreamdross, including it's strings.  Primitive look, devastating power.",
```
Comment (TheShadowFerret) [inline data/mods/Xedra_Evolved/items/range.json:9]:
```suggestion
    "description": "A six-foot bow, forged entirely from dreamdross, including its strings.  Primitive look, devastating power.",
```

File: data/mods/Xedra_Evolved/items/range.json [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+    "melee_damage": { "bash": 10 }
+  },
+  {
+    "type": "AMMO",
+    "id": "arrow_dreamforged",
+    "name": { "str": "dreamforged arrow" },
+    "symbol": "=",
+    "color": "light_gray",
+    "looks_like": "arrow_metal_sharpened_fletched",
+    "description": "A dreamforged arrow with razor sharp tip.",
```
Comment (TheShadowFerret) [inline data/mods/Xedra_Evolved/items/range.json:60]:
```suggestion
    "description": "A dreamforged arrow with razor-sharp tip.",
```

File: data/mods/Xedra_Evolved/items/range.json [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+  },
+  {
+    "id": "dreamforged_crossbow",
+    "looks_like": "crossbow",
+    "type": "GUN",
+    "symbol": "(",
+    "color": "light_gray",
+    "name": { "str": "compound crossbow" },
+    "reload_noise_volume": 4,
+    "description": "A medieval crossbow, forged entirely from dreamdross, including it's strings.  Probably not the best weapon in modern world, it still capable to do devastating damage while being completely silent",
```
Comment (TheShadowFerret) [inline data/mods/Xedra_Evolved/items/range.json:83]:
```suggestion
    "description": "A medieval crossbow, forged entirely from dreamdross, including its strings.  Probably not the best weapon in modern world, but still capable to do devastating damage while being completely silent.",
```

File: data/mods/Xedra_Evolved/items/spell_learning_items/spell_learning_items_dreamer.json [hunk 7]
```diff
@@ -217,7 +206,21 @@
...
     "color": "black",
-    "//scannable": [ false, "not a book, but shouldn't be scannable" ],
     "use_action": { "type": "learn_spell", "spells": [ "summon_winch" ] }
+  },
+  {
+    "id": "artifact_karma_arms",
+    "type": "GENERIC",
+    "category": "artifacts",
+    "name": "odd circle",
+    "description": "Circular unit of energy, with few another circles that almost make a connection around the center",
```
Comment (TheShadowFerret) [inline data/mods/Xedra_Evolved/items/spell_learning_items/spell_learning_items_dreamer.json:216]:
```suggestion
    "description": "Circular unit of energy, surrounded by a few other circles that almost make a connection around the center.",
```
image
So it looks like this?

File: data/mods/Xedra_Evolved/items/spell_learning_items/spell_learning_items_eater.json [hunk 8]
```diff
@@ -295,7 +278,21 @@
...
     "color": "dark_gray",
-    "//scannable": [ false, "not a book, but shouldn't be scannable" ],
     "use_action": { "type": "learn_spell", "spells": [ "spell_unbreakable" ] }
+  },
+  {
+    "id": "artifact_point_blank",
+    "type": "GENERIC",
+    "category": "artifacts",
+    "name": { "str": "fist" },
+    "description": "A solid, full size cast of human fist, made out of some gray metal.  Looks menacing.",
```
Comment (TheShadowFerret) [inline data/mods/Xedra_Evolved/items/spell_learning_items/spell_learning_items_eater.json:288]:
```suggestion
    "description": "A solid, full size cast of a human fist, made out of some gray metal.  Looks menacing.",
```

File: data/mods/Xedra_Evolved/player/proficiencies.json [hunk 9]
```diff
@@ -10,5 +10,58 @@
...
+    "id": "prof_dreamsmithing",
+    "name": "Dreamsmithing",
+    "description": "Proficiencies for using your magic abilities to cast items with thaumaturgical abilities."
+  },
+  {
+    "type": "proficiency",
+    "id": "prof_dreamsmithing",
+    "category": "prof_dreamsmithing",
+    "name": { "str": "Basics of Dreamsmithing" },
+    "description": "You know how you use your mind, and transform dreamdross into basic shapes.",
```
Comment (TheShadowFerret) [inline data/mods/Xedra_Evolved/player/proficiencies.json:25]:
```suggestion
    "description": "You know how you use your mind to transform dreamdross into basic shapes.",
```

File: data/mods/Xedra_Evolved/player/proficiencies.json [hunk 10]
```diff
@@ -10,5 +10,58 @@
...
+    "default_time_multiplier": 1.5,
+    "default_skill_penalty": 0,
+    "time_to_learn": "2 h"
+  },
+  {
+    "type": "proficiency",
+    "id": "prof_dreamsmithing_weapon",
+    "category": "prof_dreamsmithing",
+    "name": { "str": "Dreamsmithing of Weapons" },
+    "description": "Your knowledge grows, and you know how to shape dreamdross into a weapon of terrible power.",
```
Comment (TheShadowFerret) [inline data/mods/Xedra_Evolved/player/proficiencies.json:36]:
```suggestion
    "description": "Your knowledge has grown, and you know how to shape dreamdross into a weapon of terrible power.",
```

File: data/mods/Xedra_Evolved/player/proficiencies.json [hunk 11]
```diff
@@ -10,5 +10,58 @@
...
+    "default_time_multiplier": 1.5,
+    "default_skill_penalty": 0,
+    "time_to_learn": "4 h"
+  },
+  {
+    "type": "proficiency",
+    "id": "prof_dreamsmithing_armor",
+    "category": "prof_dreamsmithing",
+    "name": { "str": "Dreamsmithing of Armor" },
+    "description": "Your knowledge grows, and you know how to shape dreamdross into an armor of indestructible properties.",
```
Comment (TheShadowFerret) [inline data/mods/Xedra_Evolved/player/proficiencies.json:48]:
```suggestion
    "description": "Your knowledge has grown, and you know how to shape dreamdross into an indestructible armor.",
```

File: data/mods/Xedra_Evolved/spells/dreamer_spells.json [hunk 12]
```diff
@@ -524,8 +524,32 @@
...
-    "final_casting_time": { "math": [ "spell_time(u_val('time: 4 h'))" ] },
+    "base_casting_time": { "math": [ "spell_time(u_val('time: 2 h'))" ] },
+    "final_casting_time": { "math": [ "spell_time(u_val('time: 2 h'))" ] },
     "spell_class": "DREAMER"
+  },
+  {
+    "id": "spell_karma_arms",
+    "type": "SPELL",
+    "name": "Karma Arms",
+    "description": "A four transparent arms grow out of your body.  You have a full control over them, and who knows what power they contain.",
```
Comment (TheShadowFerret) [inline data/mods/Xedra_Evolved/spells/dreamer_spells.json:535]:
```suggestion
    "description": "Four transparent arms grow out of your body.  You have full control over them, and who knows what power they contain.",
```

File: data/mods/Xedra_Evolved/spells/dreamer_spells.json [hunk 13]
```diff
@@ -524,8 +524,32 @@
...
+    "//": "obvious reference for submachine, albeit no portals",
```
Comment (TheShadowFerret) [inline data/mods/Xedra_Evolved/spells/dreamer_spells.json:536]: What does this mean 🤔

File: data/mods/Xedra_Evolved/spells/eater_spells.json [hunk 14]
```diff
@@ -514,5 +514,49 @@
     "min_duration": { "math": [ "spell_time(u_val('time: 2 m'))" ] },
     "max_duration": { "math": [ "spell_time(u_val('time: 2 h'))" ] },
     "duration_increment": { "math": [ "spell_time(u_val('time: 2 m'))" ] }
+  },
+  {
+    "id": "point_blank",
+    "type": "SPELL",
+    "name": "Point Blank",
+    "description": "Straight hit with your arm.  Using additional energy, you probably can break a wall with such powerful strike.",
```
Comment (TheShadowFerret) [inline data/mods/Xedra_Evolved/spells/eater_spells.json:522]:
```suggestion
    "description": "Straight hit with your arm.  Using additional energy, you probably can break a wall with such a powerful strike.",
```

File: data/mods/Xedra_Evolved/items/spell_learning_items/spell_learning_items_dreamer.json (hunk 7 above)
Comment (GuardianDll) [reply data/mods/Xedra_Evolved/items/spell_learning_items/spell_learning_items_dreamer.json:216]: image

File: data/mods/Xedra_Evolved/spells/dreamer_spells.json (hunk 13 above)
Comment (GuardianDll) [reply data/mods/Xedra_Evolved/spells/dreamer_spells.json:536]: <https://submachine.fandom.com/wiki/Submachine_Wiki>

Comment (GuardianDll) [reply data/mods/Xedra_Evolved/spells/dreamer_spells.json:536]:
```suggestion
    "//": "obvious reference for "Submachine" by Mateusz Skutnik, albeit no portals",
```

File: data/mods/Xedra_Evolved/spells/dreamer_spells.json [hunk 15]
```diff
@@ -524,8 +524,32 @@
...
+    "base_casting_time": { "math": [ "spell_time(u_val('time: 2 h'))" ] },
+    "final_casting_time": { "math": [ "spell_time(u_val('time: 2 h'))" ] },
     "spell_class": "DREAMER"
+  },
+  {
+    "id": "spell_karma_arms",
+    "type": "SPELL",
+    "name": "Karma Arms",
+    "description": "Four transparent arms grow out of your body.  You have full control over them, and who knows what power they contain.",
+    "//": "obvious reference for "Submachine" by Mateusz Skutnik, albeit no portals",
```
Comment (GuardianDll) [inline data/mods/Xedra_Evolved/spells/dreamer_spells.json:536]:
```suggestion
    "//": "obvious reference for 'Submachine' by Mateusz Skutnik, albeit no portals",
```

File: data/mods/Xedra_Evolved/effects/effects.json [hunk 16]
```diff
@@ -429,7 +429,7 @@
     "type": "effect_type",
     "name": [ "Dream Gorging" ],
     "desc": [
-      "You ate a whole dreamdross ingot.  It is too much for your body, but temporarily you have a +6 to your strength and better regeneration."
+      "You ate a whole dreamdross ingot.  It is too much for your body, but for a while you are much <color_green>stronger</color>, and <color_green>restore from traumas</color> faster"
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/effects/effects.json:432]:
```suggestion
      "You ate a whole dreamdross ingot.  It is too much for your body, but for a while you are much <color_green>stronger</color>, and <color_green>recover from traumas</color> faster"
```

File: data/mods/Xedra_Evolved/effects/effects.json [hunk 17]
```diff
@@ -452,7 +452,7 @@
     "type": "effect_type",
     "id": "spell_stamina_eff",
     "name": [ "Circulatory" ],
-    "desc": [ "You can't get out of breath." ],
+    "desc": [ "You can't get out of <color_green>breath</color>." ],
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/effects/effects.json:455]:
```suggestion
    "desc": [ "You can't run out of <color_green>breath</color>." ],
```

File: data/mods/Xedra_Evolved/effects/effects.json [hunk 18]
```diff
@@ -589,12 +594,53 @@
...
+    "show_intensity": false,
+    "show_in_info": false,
+    "enchantments": [ { "emitter": "emit_blood_karma" } ]
+  },
+  {
+    "type": "mutation",
+    "id": "karma_arms",
+    "name": { "str": "Karma" },
+    "points": 1,
+    "description": "An ethereal hands appear behind you back, making you looks like an ancient god.",
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/effects/effects.json:620]:
```suggestion
    "description": "Ethereal hands appear behind you back, making you looks like an ancient god.",
```

File: data/mods/Xedra_Evolved/effects/effects.json [hunk 19]
```diff
@@ -589,12 +594,53 @@
...
+        ]
+      }
+    ]
+  },
+  {
+    "type": "effect_type",
+    "id": "spell_karma_arms_eff",
+    "name": [ "" ],
+    "desc": [ "" ],
+    "apply_message": "An ethereal, transparent arms grow from your shoulders, and <color_cyan>glow<color>.",
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/effects/effects.json:641]:
```suggestion
    "apply_message": "Ethereal, transparent arms grow from your shoulders, and <color_cyan>glow<color>.",
```

File: data/mods/Xedra_Evolved/eocs/dreamsmith.json [hunk 20]
```diff
@@ -17,7 +17,16 @@
           "dreamdross_lump",
           "forged_dreamstuff_ingot"
         ],
-        "type": "recipe"
+        "type": "recipe",
+        "true_eocs": {
+          "id": "EOC_MESSAGE_SCORE_TIMER",
+          "effect": [
+            {
+              "u_message": "Something sparked in your head, and you realise interesting property of dreamdross.  Maybe you can forge something new?",
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/eocs/dreamsmith.json:25]:
```suggestion
              "u_message": "Something sparked in your head, and you realize an interesting property of dreamdross.  Maybe you can forge something new?",
```

File: data/mods/Xedra_Evolved/eocs/dreamsmith.json [hunk 21]
```diff
@@ -79,17 +88,27 @@
...
-      },
-      {
-        "u_message": "A brilliant idea about the dreamdross properties hit your head, and you think you know how to utilize one now.",
-        "type": "info"
+        "type": "recipe",
+        "true_eocs": {
+          "id": "EOC_DREAMSMITH_MESSAGE_POSITIVE",
+          "effect": [
+            {
+              "u_message": "A brilliant idea about the dreamdross properties hit your head, and you think you know how to utilize one now.",
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/eocs/dreamsmith.json:102]:
```suggestion
              "u_message": "A brilliant idea about the properties of dreamdross  hits your head, and you think you know how to utilize it now.",
```

File: data/mods/Xedra_Evolved/items/range.json [hunk 22]
```diff
@@ -0,0 +1,135 @@
...
+    "melee_damage": { "bash": 10 }
+  },
+  {
+    "type": "AMMO",
+    "id": "arrow_dreamforged",
+    "name": { "str": "dreamforged arrow" },
+    "symbol": "=",
+    "color": "light_gray",
+    "looks_like": "arrow_metal_sharpened_fletched",
+    "description": "A dreamforged arrow with razor-sharp tip.",
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/items/range.json:60]:
```suggestion
    "description": "A dreamforged arrow with a razor-sharp tip.",
```

File: data/mods/Xedra_Evolved/recipes/dreamforged_weapon.json [hunk 23]
```diff
@@ -74,7 +74,7 @@
     "components": [ [ [ "forged_dreamstuff_ingot", 5 ] ] ],
     "difficulty": 4,
     "flags": [ "SECRET" ],
-    "time": "21 h"
+    "time": "7 h"
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/recipes/dreamforged_weapon.json:77]: Did you decide the ones after this didn't need proficiencies?

File: data/mods/Xedra_Evolved/spells/spell_eocs.json [hunk 24]
```diff
@@ -2,12 +2,12 @@
...
     "id": "EOC_DREAMER_ARTIFACT_SPAWN",
-    "condition": { "math": [ "u_val('vitamin', 'name:dreamer_vit')", ">=", "800" ] },
+    "condition": { "math": [ "u_val('vitamin', 'name:dreamer_vit')", ">=", "300" ] },
     "effect": [
       { "u_spawn_item": "dreamer_artifact", "use_item_group": true },
-      { "math": [ "u_val('vitamin', 'name:dreamer_vit')", "-=", "800" ] },
+      { "math": [ "u_val('vitamin', 'name:dreamer_vit')", "-=", "300" ] },
       {
-        "u_message": "You pulled your powers into item, and something outside instigate item in your hand.",
+        "u_message": "You pulled your powers into a matter, and something outside instigate item in your hand.",
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/spells/spell_eocs.json:10]:
```suggestion
        "u_message": "You pulled your powers into this world and something from outside placed an object in your hand.",
```

**Review by Maleclypse — COMMENTED** [review]
Good stuff :)

File: data/mods/Xedra_Evolved/recipes/dreamforged_weapon.json (hunk 23 above)
Comment (GuardianDll) [reply data/mods/Xedra_Evolved/recipes/dreamforged_weapon.json:77]: Profs are inherited by `"copy-from": "dreamforged_longsword_ench",`

**Pull request merged**